documentation/src/main/markdown/packagemgrs/conan.md

```diff
@@ -61,40 +62,20 @@ In the zlib example:
 * channel=_ (by default)
 * recipe_revision=1a67b713610ae745694aa4df1725451d
 
-[solution_name] constructs a KB external ID for namespace "conan" using these fields as follows:
+[solution_name] constructs a [blackduck_kb] external ID for namespace "conan" using these fields as follows:
 ```
 <name>/<version>@<user>/<channel>#<recipe_revision>
 ```
 
-## Package revision matching
```

Comment:
I removed this section because when I tested the pre-existing behaviour with setting `detect.conan.attempt.package.revision.match` to `true`, I got no matches from KB.

Given that it has been several years since Conan support was introduced and package revision support is still not implemented it seems to me like it's better to not advertise this functionality.

Additionally, I am going to look into deprecating this setting from Detect.

Reply:
There is a QA test that uses detect.conan.attempt.package.revision.match=true and gets matches. However, there are slightly more matches when detect.conan.attempt.package.revision.match=false.